bundles/org.openhab.binding.digitalstrom/src/main/java/org/openhab/binding/digitalstrom/internal/handler/BridgeHandler.java

```diff
@@ -26,6 +26,7 @@
 import java.util.concurrent.TimeUnit;
 
 import org.apache.commons.lang.StringUtils;
+import org.eclipse.jdt.annotation.NonNull;
```

Comment:
`NonNull` annotations should not be needed. It means the `NonNullByDefault` on class level is missing, which is the preferred way to annotate. I can image it's a lot more work to add `NonNullByDefault` to the class, so I don't expect to add that, but I also don't think you should add the `NonNull` annotations as a kind of half fix. So it's better to have no annotations than to have a wrong half solution.

Reply:
Removed